### DISCUSSION TITLE
How to save the detection result?

### DISCUSSION THREAD

**Discussion by Yoooss** [open-mmlab/mmselfsup#666] [How-to]
I used the command "bash ./tools/benchmarks/mmdetection/mim_dist_train_fpn.sh configs/benchmarks/mmdetection/voc0712/faster_rcnn_swin_fpn_voc0712ls_iter_grid_num.py /home/ls/mmselfsupv1/work_dirs/selfsup/densecl_resnet50_8xb32-coslr-200e_in1k_swin_pred_grid_num/20221213_161335_grid1/epoch_200.pth 1" to train the pretrained model on the object detection dowmstream task. And I got the mAP result.
Then I want to save the detection result ,but I didn't find relevant tutorials. Mat I get some help?
I want to see the detection result in visualization .I want to save the "result.jpg" which has the detection bouding box on the object.
I'm sorry for my pool english.Hope I have expressed my question clearly.May I get some help?
Thanks a lot.

**Comment by Yoooss** [comment]
And the version of mmselfsup I use  is mmselfsupv1.

**Comment by fangyixiao18** [comment]
You could check some docs in MMDetection. 
Here are some examples, which may help.
https://github.com/open-mmlab/mmdetection/blob/master/demo/inference_demo.ipynb
https://github.com/open-mmlab/mmdetection/blob/master/docs/en/useful_tools.md
